### PR TITLE
Add a --keep-message option to avoid editing pull request messages.

### PR DESCRIPTION
**Pull request by Flameeyes**
This is to address issue #108 — it streamlines a bit refreshing an
already-created pull request.

### CODE REVIEW

**Comment by Flameeyes** [comment]
Note: I'm about to add a commit to this that is not relevant to the issue at all, to make sure it works right ;)

**Comment by Flameeyes** [comment]
Yeah this worked fine for my needs.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[build-system]
```
Comment (sileht) [inline pyproject.toml:1]: Looks unrelated to the change.

Comment (sileht) [reply pyproject.toml:1]: `use_scm_version=True` in the new configuration

Comment (Flameeyes) [reply pyproject.toml:1]: It is unrelated, see my [previous comment](https://github.com/Mergifyio/git-pull-request/pull/109#issuecomment-612661014) about trying this out :)

As for `use_scm_version` — that is not needed when using `pyproject.toml`: https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage

**Comment by jd** [comment]
Can we tweak the build system in a different PR?

**Comment by Flameeyes** [comment]
> Can we tweak the build system in a different PR?

Done.
